src/runtime/sgx/untrusted/sgx_module.cc

```diff
@@ -177,10 +198,20 @@ void tvm_ocall_packed_func(const char* name,
 
 // Allocates space for return values. The returned pointer is only valid between
 // successive calls to `tvm_ocall_reserve_space`.
-void* tvm_ocall_reserve_space(size_t num_bytes) {
-  static thread_local std::vector<uint64_t> buf;
-  buf.reserve(num_bytes);
-  return buf.data();
+void* tvm_ocall_reserve_space(size_t num_bytes, size_t alignment) {
+  static TVMContext ctx = { kDLCPU, 0 };
+  static thread_local void* buf;
+  static thread_local size_t buf_size;
```

Comment:
initialize buf_size, otherwise behavior is undefined.

Reply:
According to [the C++11 spec (p191)](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2012/n3337.pdf), scalar types with static or thread_local duration are zero initialized.

Reply:
OK, I stil still better to be explicit so the code is more readable